src/kata1.fizzBuzz.js

```diff
@@ -1,3 +1,18 @@
-const fizzBuzz = number => {};
+const fizzBuzz = number => {
```

Comment:
I try to avoid chaining ternaries as they get harder to read...

you could have consider `if`, or `switch` here.